Combinatorial_map/doc/Combinatorial_map/Concepts/GenericMap.h

```diff
@@ -976,6 +1008,17 @@ This is possible if `d1`\f$ \neq \f$ `d2` and `d1` can be reached from `d2` by u
 */
 bool is_insertable_cell_1_in_cell_2(Dart_const_descriptor d1, Dart_const_descriptor d2);
 
+/*!
+Returns true iff it is possible to insert a 1-cell in the generic map between `d1` and `d2`.
+
+This is possible if `d1`\f$ \neq \f$ `d2` and `d1` can not be reached from `d2` by using some `previous` and `next` calls.
```

Comment:
We had a discussion about mathmode vs code for not equal.   I prefer code.

Reply:
you want <code>!=</code> ?

Reply:
Indeed I use \f$ \neq \f$ in many other places; thus it is probably better to keep it.